kano/utils/hardware.py

```diff
@@ -10,8 +10,28 @@
 from kano.utils.file_operations import read_file_contents_as_lines
 
 
-def detect_kano_keyboard():
+# "performance" scores for RPi boards
+RPI_A_SCORE = 1000
+RPI_A_PLUS_SCORE = 1000
+RPI_B_SCORE = 2000
+RPI_B_PLUS_SCORE = 2000
+RPI_ZERO_SCORE = 3000
```

Comment:
Hmm. Actually the CM might be better than the pi2 under some circumstances :-)
I guess it's okay for now since all this is internal to this module and we are only using RPI_2_B_SCORE elsewhere anyway.